src/main/java/frc/utils/dashboard/LoggedTableBoolean.java

```diff
@@ -0,0 +1,44 @@
+package frc.utils.dashboard;
+
+import edu.wpi.first.networktables.NetworkTable;
+import edu.wpi.first.networktables.NetworkTableInstance;
+import edu.wpi.first.networktables.NetworkTableValue;
+import org.littletonrobotics.junction.Logger;
+import org.littletonrobotics.junction.networktables.LoggedDashboardInput;
+
+public class LoggedTableBoolean implements LoggedDashboardInput {
+
+    private final String key;
+    private final NetworkTable booleanTable;
+    private boolean value;
+
+    public LoggedTableBoolean(String table, String key) {
+        this(table, key, false);
+    }
+
+    public LoggedTableBoolean(String table, String key, boolean defaultValue) {
+        this.key = key;
+        this.value = defaultValue;
+        this.booleanTable = NetworkTableInstance.getDefault().getTable(table);
+
+        set(defaultValue);
```

Comment:
This, function makes the first time the value become true keep true instead of getting false until triggering the entry again.